pyuvdata/uvdata/ms.py

```diff
@@ -394,6 +397,7 @@ def read_ms(
         assert tb_field.getcol("PHASE_DIR").shape[1] == 1, message
 
         self.phase_type = "phased"
+        self.phase_center_frame = "icrs"
         # MSv2.0 appears to assume J2000. Not sure how to specifiy otherwise
```

Comment:
J2000 does indeed mean FK5 -- it's just not an allowed keyword for `phase_center_frame` (meant to add a TODO here). I should be able to fix this in the `UVData` object though, and will fix this line when I do so.

Reply:
I think "FK5" is a model and "J2000.0" is an epoch of said model that is more commonly referred to as simply "J2000".  ICRS was defined to be close to J2000, but there is a small frame bias between them that I've seen described as "of order 25 milliarcseconds", "of about 23 mas", and "Subsequent observations located the J2000.0 mean pole at ICRS [X,Y ] coordinates [-0.016617", -0.0068192"]".  Those latter two are from the (very excellent, btw) [SOFA Tools for Earth Attitude](http://www.iausofa.org/sofa_pn_c.pdf) "cookbook".

Reply:
Yeah, I was just about to write -- after digging into the guts of the astrometry, I realize that I'm conflating two separate concepts here. J2000 and other epoch values talking about a particular time, but are _also_ used as a shorthand way of referring to the position of the equinox at said time (which sets the "zero-point" for the RA). You _can_ use Julian dates (e.g., J2021.0) to refer to the Bessel-Newcomb (FK4) equinox, but for obvious reasons, that's not typical.

For most applications, the only reason you care about the epoch is so that you can get the right value of the equinox (following account for precession, nutation, etc), and so for a not-rotating frame like ICRS, the epoch value becomes superfluous. The one case I stumbled across where this isn't true is with proper motion, where the epoch defines _when_ those coordinates are specified; e.g., two sources with the same amount of proper motion but different epochs defined will have different distances from their 'reference' RA/Dec position.

Given the recent keyword discoveries in UVFITS, I'm tempted to first look for said keywords, then trying to default to a sensible default if none are discovered. As @david-macmahon above points out, the difference between ICRS and FK5 @ J2000.0 is small enough to be negligible for most cases (with a median error of about 15 mas, and a max error of about 25 mas), so even in the case where one has misidentified the frame, the impact would be small.

Reply:
One more annoying wrinkle in all of this -- CASA has a coordinate reference frame called 'J2000' that _is_ actually FK5 at Julian epoch J2000.0. CASA doesn't seem to actually support the concept of an equinox other than the typically used one, and so in this one specific instance, J2000 means both. This is, presumably, why we can't have nice things.